### DISCUSSION TITLE
Swagger UI crashes with RangeError: Maximum call stack size exceeded` for nested 15K-item response

### DISCUSSION THREAD

**Discussion by mmlynarik** [fastapi/fastapi#8830] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# app.py
from fastapi import FastAPI
import uvicorn

app = FastAPI()

@app.get('/api/data', tags=['Data'])
def map_data():
    mapping = {}
    for i in range(15000):
        mapping[i] = [
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988,
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988,
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988
        ]
    return mapping


if __name__ == '__main__':
    uvicorn.run("app:app", host="127.0.0.1", port=5000)
```

### Description

* Open the browser and call the endpoint /data
* The web UI starts to get the JSON, but after a several tens of seconds, it finishes with error `RangeError: Maximum call stack size exceeded` 

### Operating System

Linux

### Operating System Details

WSL Ubuntu 20.04

### FastAPI Version

0.65.2

### Python Version

3.8.5

### Additional Context

The same test case when run using Connexion framework (which also uses Swagger UI) finishes correctly without error:

```python

# app.py
import connexion

def map_data():
    mapping = {}
    for i in range(15000):
        mapping[i] = [
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988,
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988,
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988
        ]
    return mapping

app = connexion.App(__name__, specification_dir="./")
application = app.app  # expose global WSGI application object
app.add_api("swagger.yml")


# swagger.yml
swagger: "2.0"
info:
  description: Interactive Swagger UI for Data Mappings
  version: "1.0"
  title: Swagger RESTful API
consumes:
  - application/json
produces:
  - application/json

basePath: /

paths:
  /data:
    get:
      operationId: api.app.map_data
      tags:
        - Map data
      summary: Map data
      description: Map data
      responses:
        200:
          description: Successfully read mappings for all data fields
          schema:
            type: object

**Comment by Mause** [comment]
This sounds like an issue with swagger ui, not fastapi?

**Comment by mmlynarik** [comment]
> This sounds like an issue with swagger ui, not fastapi?

Yes, I was considering also reporting it under Swagger repository (and I will definitely do it), but also wanted to get some insight here first, because when I run the same endpoint using [Connexion](https://github.com/zalando/connexion) framework, it runs without problems, see `Additional Context` section. So I suppose there might be somehow problem with specific combination of FastAPI and Swagger.

**Comment by YuriiMotov** [comment]
Current version works fine with this code example
